Building on macOS 12/m1 breaks

### Description

It is currently not possible to build the mumble client on macOS.

### Steps to reproduce

```
cd build
cmake -Dice=off -DCMAKE_BUILD_TYPE=Release ..
cmake --build . -j 4
```

### Mumble version

3960373d9

### Mumble component

Client

### OS

macOS

### Reproducible?

Yes

### Additional information

_No response_

### Relevant log output

```shell
[ 65%] Building CXX object src/mumble/CMakeFiles/mumble.dir/PAAudio.cpp.o
/Users/thegcat/Code/m1/mumble/src/mumble/GlobalShortcut_macx.mm:26:7: error: variable 'forward' set but not used [-Werror,-Wunused-but-set-variable]
        bool forward = false;
             ^
1 error generated.
make[2]: *** [src/mumble/CMakeFiles/mumble.dir/GlobalShortcut_macx.mm.o] Error 1
```


### Screenshots

_No response_

--- a/src/mumble/GlobalShortcut_macx.mm
+++ b/src/mumble/GlobalShortcut_macx.mm
@@ -119,6 +119,9 @@ CGEventRef GlobalShortcutMac::callback(CGEventTapProxy proxy, CGEventType type,
 			[pool release];
 			return nullptr;
 		}
+#else
+	// Mark forward as unused in this case
+	(void) forward;
 #endif
 	return suppress ? nullptr : event;
 }

